apps/condo/domains/common/utils/url.utils.js

```diff
@@ -40,8 +40,16 @@ function extractRootDomain (url) {
     return domain
 }
 
+function extractOrigin (url) {
```

Comment:
can you please add comment with example, what's happening here ?

Reply:
Added :)